templates/Web/Projects/VueDefault/package_postaction.json

```diff
@@ -0,0 +1,23 @@
+  },
+//{[{
+  "eslintConfig": {
+    "root": true,
+    "env": {
+      "node": true
+    },
+    "extends": [
+      "plugin:vue/essential",
+      "eslint:recommended"
+    ],
+    "rules": {},
+    "parserOptions": {
+      "parser": "babel-eslint"
+    }
+  },
+  "postcss": {
+    "plugins": {
+      "autoprefixer": {}
+    }
+  },
+//}]}
+  "browserslist": [
```

Comment:
formatting seems messed up here

Reply:
or maybe is this supposed to be inserted somewhere

Reply:
it seems messed up but this gets inserted properly, try hitting generate post packaging. The red markers are there because json doesn't support comments. is there anything specific you're refering to?

Reply:
Oh just saw your other comment yup this is meant to be inserted between a 
```
},

and the key:
```
"browserslist": [
```

It will be inserted as the first element between these two things.